go.mod

```diff
@@ -125,3 +128,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/hashicorp/terraform-exec v0.21.0 => github.com/hrmsk66/terraform-exec v0.21.0
```

Comment:
Using a fork awaiting the merge of this PR: https://github.com/hashicorp/terraform-exec/pull/446/